pip-freeze.txt

```diff
@@ -28,6 +28,7 @@ enum34==1.0.4
 flake8==2.5.4
 gunicorn==19.3.0
 hamlpy==0.82.2
+hypothesis==3.4.0
```

Comment:
Also, note that this is only a testing requirement, but casepro doesn't have a separate testing/dev requirements file, and I didn't want to add one (because it would mean updating docs and travis config, and other existing devs would have to do different things to get tests working)